ckanext/dcatapit/plugin.py

```diff
@@ -481,6 +481,13 @@ def after_show(self, context, pkg_dict):
             if df_value:
                 tmp_value = validators.parse_date(df_value, df_value)
                 if isinstance(tmp_value, datetime.date):
+                    # there are values that are noted as 17 or 0017
+                    # this may come from short date notation used with long year format
+                    # parsed in. Anyway, it's simply wrong value. This can be fixed by
+                    # increasing year by 2000 (we assume those values come from recent years)
+                    # see https://github.com/geosolutions-it/ckanext-datitrentinoit/issues/36
+                    if tmp_value.year < 1900:
```

Comment:
@cezio,

In my opinion this is a wrong assumption. We should add a try/catch block for this in order to properly manage wrong date and so avoid formatting wrong values at all. This is a most general and secure way to capture all wrong cases.

Reply:
Ok. Because that date value can't be formatted with `strftime()`, i'll use raw value. This will however lead to values displayed as:

![image](https://user-images.githubusercontent.com/116021/46545885-ddcc4900-c8c7-11e8-9bce-cf33b8977a1f.png)